brave/browser-laptop
# Option to clear Sync server data and local browser state

Fix #7275

Fix https://github.com/brave/sync/issues/51


Test Plan:
1. Spawn a clean pyramid 0. Enable Sync and add bookmarks.
2. Spawn a clean pyramid 1 (`npm run start2`). Add it to 0's Sync profile and see the bookmarks sync over.
3. Save the Sync code words. We'll use this to demonstrate data was purged.
4. In Pyramid 0, go to Preferences > Sync and click Reset data. In the console you should see "Deleting user!!".
5. Close Pyramid 1, delete its user data, and spawn it clean again. Using the saved code words, try to re-add it to 0's deleted Sync profile. No bookmarks should sync over.
6. Back in Pyramid 0, you can use Sync freshly. Create a new profile and you should get a new set of code words.

Pix
![screen shot 2017-02-16 at 16 48 00](https://cloud.githubusercontent.com/assets/521861/23188721/a41ed65a-f844-11e6-9a25-3699d65b7d30.png)
![screen shot 2017-02-16 at 16 48 15](https://cloud.githubusercontent.com/assets/521861/23188724/a72f5cac-f844-11e6-9d83-a28dacd1edb0.png)
